README.md

```diff
@@ -1,7 +1,7 @@
 # ![voila](docs/source/voila-logo.svg)
 
 [![Documentation](http://readthedocs.org/projects/voila/badge/?version=latest)](https://voila.readthedocs.io/en/latest/?badge=latest)
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/QuantStack/voila/stable?urlpath=voila%2Ftree%2Fnotebooks)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/voila-gridstack/voila/stable?urlpath=voila%2Ftree%2Fnotebooks)
```

Comment:
This looks like a typo and should be `voila-dashboards`

Reply:
Yes! Please feel free to fix in master.